poc/test.sage

```diff
@@ -61,11 +62,25 @@ if __name__ == "__main__":
     test_sswu_9mod16()
 
     print("Testing generic maps")
-    for m in (GenericEll2, GenericEll2Edw, GenericSSWU, GenericSvdW):
+    for m in (GenericEll2, GenericEll2Edw, GenericSSWU, OptimizedSSWU, GenericSvdW):
         print("Testing %s" % m.__name__)
         for _ in range(0, 32):
             m.test_random()
 
+    print("Testing SSWU map equality")
```

Comment:
This should also be tested at the `undefs` elements (preferably for curves where -1/Z is and is not square).

Reply:
This comment is still applicable.